src/components/SwitchLanguageButton.tsx

```diff
@@ -3,11 +3,16 @@ import { SwitchLanguageModal } from '../modals/SwitchLanguageModal';
 import { languageIcons } from '../I18n';
 import { Button } from './Button';
 import { cn } from '@/lib/Tailwind';
+//import '/node_modules/flag-icons/css/flag-icons.min.css';
+import { DE } from 'country-flag-icons/react/3x2';
```

Comment:
This should do the trick:

```suggestion
import { DE } from 'country-flag-icons/react/1x1';
```